test/mocha/cli.js

```diff
@@ -744,4 +744,28 @@ describe("bin/uglifyjs", function () {
             done();
         });
     });
+    it("Should work with -e (default)", function(done) {
+        var command = uglifyjscmd + " test/input/enclose/input.js -e";
```

Comment:
`--enclose`

Reply:
is command also need use --enclose or now?

Reply:
It is much easier to read and understand `--enclose` in the test code then the shorthand.

You are welcome to keep `-e` under `bin/uglifyjs`, just do **not** use it in any of the tests here.

Reply:
In command I used shorthand because I find other tests use it like `--mangle`.
Let's me update now 😄